### ISSUE TITLE
Use APPLICATION_HOST variable instead of hard-coded URLs

### ISSUE THREAD

**Issue by demery** [KislakCenter/VisualCollation#116]
A number of places use hardcoded URLs and should rely on the `APPLICATION_HOST` variable which includes the port (if needed); e.g., `vceditor.library.upenn.edu:8443`.

This change should be made in the mailer ERB files `confirmation_instructions.html.erb` and `sendApprovalStatus.html.erb`. Other mailer ERB files should be checked for hardcoded URLs. 

Any ERB file that has an `img@src` pointing to `https://vceditor.library.upenn.edu` should be changed to use the `APPLICATION_HOST` variable:

```ruby
<img src="https://#{APPLICATION_HOST}"/>
```

The `APPLICATION_HOST` variable should also be used in [`config/initializers/rails_jwt_auth.rb`](https://github.com/KislakCenter/VisualCollation/blob/development/viscoll-api/config/initializers/rails_jwt_auth.rb#L24-L30): 

```ruby
  # url used to create email link with confirmation token  
  config.confirmation_url = if Rails.env.production? then 'https://vceditor.library.upenn.edu/confirmation' else 'http://127.0.0.1:3000/confirmation' end

  # expiration time for confirmation tokens
  #config.confirmation_expiration_time = 1.day

  # url used to create email link with reset password token  
  config.reset_password_url = if Rails.env.production? then 'https://vceditor.library.upenn.edu/password' else 'http://127.0.0.1:3000/password' end
```

NB: It's a little off that the port is in the application host variable, and it would fit better in a `APPLICATION_URL`, but it's fine for now and easier than reworking the code for an `APPLICATION_HOST/_URL` system.

**Comment by lilblizzard** [comment]
- [ ] confirmation_instructions.html.erb
- [ ] reset_password_instructions.html.erb
- [ ] set_password_instructions.html.erb
- [ ] sendApprovalStatus.html.erb
- [ ] rails_jwt_auth.rb

**Comment by demery** [comment]
Addressed by change #117
